clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetchMetricsManagerTest.java

```diff
@@ -32,18 +33,19 @@
 import org.junit.jupiter.api.Test;
 
 import java.util.Map;
+import java.util.Set;
 
 import static org.apache.kafka.clients.consumer.internals.FetchMetricsManager.topicPartitionTags;
 import static org.apache.kafka.clients.consumer.internals.FetchMetricsManager.topicTags;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class FetchMetricsManagerTest {
```

Comment:
Personally, I would put `@SuppressWarnings("deprecation")` on this class because it is knowingly using deprecated methods.

Reply:
Done.

Reply:
I have a suspicion you put the suppression on `FetchMetricsManager` when I meant to put it on this test.

Reply:
I have corrected it, also I have placed the suppression on individual tests so they can be removed while removing deprecated code.